math/minuit2/inc/Minuit2/FCNGradientBase.h

```diff
@@ -38,7 +43,24 @@ class FCNGradientBase : public FCNBase {
 
    virtual std::vector<double> Gradient(const std::vector<double> &) const = 0;
 
+   virtual std::vector<double> G2ndDerivative(const std::vector<double> &) const = 0;
+
+   virtual std::vector<double> GStepSize(const std::vector<double> &) const = 0;
+
```

Comment:
same comment as before for the Capitalisation of the member functions

Reply:
Resolved by removing.